--- a/temba/orgs/views/views.py
+++ b/temba/orgs/views/views.py
@@ -24,7 +24,7 @@
 from django.db.models import F, Prefetch, Q
 from django.db.models.functions import Lower
-from django.http import HttpResponseRedirect, JsonResponse
+from django.http import HttpResponse, HttpResponseRedirect, JsonResponse
 from django.shortcuts import get_object_or_404
 from django.urls import reverse, reverse_lazy
 from django.utils import timezone
@@ -1747,7 +1747,15 @@ class Download(SpaMixin, ContextMenuMixin, OrgObjPermsMixin, SmartReadView):
         def get(self, request, *args, **kwargs):
             if str_to_bool(request.GET.get("raw", 0)):
                 export = self.get_object()
-
-                return HttpResponseRedirect(export.get_raw_url())
+                url = export.get_raw_url()
+
+                # if our URL doesn't have the content disposition set (local storage), we can set it ourselves
+                # using X-Accel-Redirect
+                if "ResponseContentDisposition" not in url:
+                    response = HttpResponse()
+                    response["X-Accel-Redirect"] = url
+                    response["Content-Disposition"] = f'attachment; filename="{export._get_download_filename()}"'
+                    return response
+
+                return HttpResponseRedirect(url)
 
             return super().get(request, *args, **kwargs)
