--- /tmp/p_orig	2026-04-09 20:37:54.433866876 +0200
+++ /home/frappe/frappe-bench/apps/frappe/frappe/public/js/frappe/form/form.js	2026-04-09 20:36:28.668681475 +0200
@@ -1979,7 +1979,7 @@
 
 				return `
 						<a class="indicator ${get_color(doc || {})}"
-							href="/desk/${frappe.router.slug(df.options)}/${escaped_name}"
+							href="${(frappe.router && frappe.router._subpath_prefix) || ""}/desk/${frappe.router.slug(df.options)}/${escaped_name}"
 							data-doctype="${df.options}"
 							data-name="${frappe.utils.escape_html(value)}">
 							${label}
@@ -2289,7 +2289,7 @@
 						secondary = `
 						</div>
 						<div class="col-md-6">
-							<a href='/desk/submission-queue?ref_doctype=${encodeURIComponent(
+							<a href='${(frappe.router && frappe.router._subpath_prefix) || ""}/desk/submission-queue?ref_doctype=${encodeURIComponent(
 								this.doctype
 							)}&ref_docname=${encodeURIComponent(this.docname)}'>${__(
 							"All Submissions"
@@ -2300,7 +2300,7 @@
 					let html = `
 					<div class="row">
 						<div class="${div_class}">
-							<a href='/desk/submission-queue/${r.message.latest_submission}'>${submission_label} (${r.message.status})</a>${secondary}
+							<a href='${(frappe.router && frappe.router._subpath_prefix) || ""}/desk/submission-queue/${r.message.latest_submission}'>${submission_label} (${r.message.status})</a>${secondary}
 						</div>
 					</div>
 					`;
